package.json

```diff
@@ -24,12 +24,25 @@
     "test:watch": "jest --watch --verbose"
   },
   "dependencies": {
+    "@emotion/react": "11.9.3",
+    "@emotion/styled": "11.9.3",
+    "@leafygreen-ui/button": "16.0.0",
+    "@leafygreen-ui/emotion": "^4.0.0",
+    "@leafygreen-ui/leafygreen-provider": "2.3.2",
+    "@leafygreen-ui/palette": "3.4.0",
+    "@leafygreen-ui/select": "7.0.0",
+    "@leafygreen-ui/text-input": "10.1.0",
+    "@leafygreen-ui/toggle": "8.0.2",
+    "@leafygreen-ui/typography": "13.1.1",
```

Comment:
Does this feel like the right PR in which to add fonts? Feel free to say no bc Friday EOD + your vacation next week 😁 If not it looks like we should open a ticket!

Reply:
I'll open a ticket! 🙏